controllers/openstackdataplanerole_controller.go

```diff
@@ -264,8 +272,8 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 		// Return when any condition is not ready, otherwise set the role as
 		// deployed.
 		if len(novaReadyConditionsTrue) < len(nodes.Items) {
-			r.Log.Info("Not all NovaExternalCompute ReadyConditions are true")
-			return result, nil
+			r.Log.Info("Not all NovaExternalCompute ReadyConditions are true.")
```

Comment:
As a future enhancement we could log which NovaExternalCompute is not ready to easy the troubleshooting.